src/components/capture/ExpressCreate/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { authenticatedRoutes } from 'src/app/routes';
+import CaptureGenerateButton from 'src/components/capture/GenerateButton';
+import {
+    useEditorStore_id,
+    useEditorStore_persistedDraftId,
+    useEditorStore_queryResponse_mutate,
+    useEditorStore_setId,
+} from 'src/components/editor/Store/hooks';
+import EntityCreateExpress from 'src/components/shared/Entity/Create/Express';
+import EntityToolbar from 'src/components/shared/Entity/Header';
+import { MutateDraftSpecProvider } from 'src/components/shared/Entity/MutateDraftSpecContext';
+import useValidConnectorsExist from 'src/hooks/connectors/useHasConnectors';
+import useDraftSpecs from 'src/hooks/useDraftSpecs';
+import usePageTitle from 'src/hooks/usePageTitle';
+import { CustomEvents } from 'src/services/types';
+import { useDetailsFormStore } from 'src/stores/DetailsForm/Store';
+import WorkflowHydrator from 'src/stores/Workflow/Hydrator';
+import { MAX_DISCOVER_TIME } from 'src/utils/misc-utils';
+
+export default function ExpressCaptureCreate() {
+    usePageTitle({
+        header: authenticatedRoutes.express.captureCreate.new.title,
+        headerLink:
+            'https://docs.estuary.dev/guides/create-dataflow/#create-a-capture',
+    });
+
+    const entityType = 'capture';
```

Comment:
This should probably just come from the `entity type` hook. If it has to be hardcoded then it should be outside the component.